keep/rulesengine/rulesengine.py

```diff
@@ -156,6 +157,23 @@ def _run_cel_rules(
 
                                 send_created_event = incident.is_visible
 
+                            incident_id = incident
```

Comment:
just a side note: i'll like it more if we use the retry mechanism as a standard

Reply:
Yeah, I will think about total refactoring